src/core/sharedOptions.ts

```diff
@@ -48,7 +48,7 @@ export type LifeCycleEventsMap = {
   'response:bypass': [
     args: {
       response: Response
-      request: Request
+      request: Request | undefined
```

Comment:
If that's the case, it's a bug. We shouldn't accommodate it. I will look into this once I have a moment. Thanks for bringing this to my attention!

Reply:
Hi,

What do we do until you fix this bug. Should we roll back to an older version?

Reply:
@ymarios-int this is a comment chain for a line of code.  You can ask this on the issue or commit.  I am not sure when or if this was introduced.  So you can look at the history to see what may have happened.   It's only been a few days so I would just wait a bit.  you can subscribe to the issue,

Reply:
@ymarios-int looks to have been introduced in 2.1.5 so I just downgraded to "msw": "2.1.4" for now.  I am not sure if there will be other issue but I figured I'll share.

It was fixing a pretty important fixme it seems

https://github.com/mswjs/msw/commit/bad537f1fce7511f0a7258c0216a9ad49fbc27b8